schema/generate_docs/tablerowgen.py

```diff
@@ -0,0 +1,360 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from abc import ABC
+from typing import Generator
+
+from pydantic import BaseModel
+
+from ..metaschema.meta_schema_model import AvdSchemaBool, AvdSchemaDict, AvdSchemaInt, AvdSchemaList, AvdSchemaStr
+from ..metaschema.resolvemodel import resolve_model
+
+
+def get_table_rows(
```

Comment:
should not it be some function of a AvdSchemaType as well?

Like

```
AvdSchemaBool.get_table_rows()
```

Reply:
I have gone back and forth on this a couple of times, and I decided to keep it outside the main schema class, since we will have many of these converters that will only be needed for development workflows, where the main classes may be needed for ongoing validation.
Table
Yaml
Pydantic
Jsonschema? (the builtin jsonschema generator from pydantic does not handle contrained types)